lib/webapp.js

```diff
@@ -847,12 +847,13 @@ const staticHandlers = {
  *  This is passed to every other service of the plugin, so that 
  *  the service can be called by other services under the plugin
  */
-function WebServiceHandle(urlPrefix, environment) {
+function WebServiceHandle(urlPrefix, environment, isAgentService) {
   this.urlPrefix = urlPrefix;
   if (!environment.loopbackConfig.port) {
     installLog.severe(`ZWED0003E`, loopbackConfig); //installLog.severe(`loopback configuration not valid,`,loopbackConfig, `loopback calls will fail!`);
   }
   this.environment = environment;
+  this.isAgentService = isAgentService;
 }
 WebServiceHandle.prototype = {
   constructor: WebServiceHandle,
```

Comment:
The comment below can be updated to include turning off HTTP call to localhost by setting node.internalRouting=true in the server config.

Reply:
Below line 856